config/config.go

```diff
@@ -216,13 +222,13 @@ func ReadTerragruntConfig(terragruntOptions *options.TerragruntOptions) (*Terrag
 
 // Parse the Terragrunt config file at the given path. If the include parameter is not nil, then treat this as a config
 // included in some other config file when resolving relative paths.
-func ParseConfigFile(configPath string, terragruntOptions *options.TerragruntOptions, include *IncludeConfig) (*TerragruntConfig, error) {
-	configString, err := util.ReadFileAsString(configPath)
+func ParseConfigFile(filename string, terragruntOptions *options.TerragruntOptions, include *IncludeConfig) (*TerragruntConfig, error) {
```

Comment:
This same string was being passed along through a bunch of different functions, and halfway through it switched from `configPath` to `filename`.  As someone new to the codebase it was confusing, since it didn't actually change at all.  So this is just making the name consistent.

Reply:
I can always revert this change if you'd like.

Reply:
No worries, was just checking that this wasn't due to some bug or other issue